submissions/nhunzaker/test/server.test.js

```diff
@@ -0,0 +1,42 @@
+var Server = require('../lib/server')
+var assert = require('assert')
+
+describe('Server', function() {
+
+  context('When Server.init is called at a given port', function() {
+    var PORT = 9000
+
+    beforeEach(function (next) {
+      this.server = Server.init(PORT, next)
```

Comment:
I saw it used in a couple of testing examples (not specific to Hapi). I think it's a little less cumbersome, but it definitely makes it impossible to use fat arrows.

I'll keep using it and see how it feels on more elaborate tests.